cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -257,3 +262,63 @@ def test_open_url_availability(appliance):
         view.add_button.click()
         view.flash.assert_message("URL can be opened only by buttons for a single entity")
     view.cancel_button.click()
+
+
+@pytest.mark.provider([VMwareProvider], override=True, scope="function", selector=ONE_PER_TYPE)
+@pytest.mark.uncollectif(
```

Comment:
note for reviewers: no bug for 5.9 has been cloned yet. 

@digitronik I think using `@pytest.mark.ignore_stream("5.9")' would be better here

Reply:
> note for reviewers: no bug for 5.9 has been cloned yet.
> 
> @digitronik I think using `@pytest.mark.ignore_stream("5.9")' would be better here

@jawatts `ignore_stream` option good if test is `not applicable for specific version`. Its will not going to provide specific reason. Means if feature comes in `5.10`  and I don't want to run that test against `5.9` then this option  is `better` but not good to use it when `uncollected`  is `temporary`. 

> I'd vote for `pytest.mark.meta(blockers=[BZ(1646905, forced_streams=['5.9'])])`

@mshriver your `vote` always valuable :)... I was selected same option initially but after that I understand one behavior of `BZ` options. I logged this BZ against `5.9` but its fixed for `5.10`. so `BZ` will not `uncollect` it.

After understanding both scenarios. I selected `uncollefctif` as  its providing me proper `reason` attribute.  which  will remind me every time about this BZ.